Exiled.Loader/Loader.cs

```diff
@@ -347,23 +352,22 @@ private static bool CheckPluginRequiredExiledVersion(IPlugin<IConfig> plugin)
             var requiredVersion = plugin.RequiredExiledVersion;
             var actualVersion = Version;
 
-            // Check Major version
-            // It's increased when an incompatible API change was made
-            if (requiredVersion.Major != actualVersion.Major)
+            // Check the version
+            if (requiredVersion != actualVersion)
             {
-                // Assume that if the Required Major version is greater than the Actual Major version,
+                // Assume that if the Required version is greater than the Actual version,
                 // Exiled is outdated
-                if (requiredVersion.Major > actualVersion.Major)
+                if (requiredVersion > actualVersion)
                 {
-                    Log.Error($"You're running an older version of Exiled ({Version.ToString(3)})! {plugin.Name} won't be loaded! " +
-                              $"Required version to load it: {plugin.RequiredExiledVersion.ToString(3)}");
+                    Log.Error($"You're running an older version of Exiled ({Version})! {plugin.Name} won't be loaded! " +
+                              $"Required version to load it: {plugin.RequiredExiledVersion}");
 
                     return true;
                 }
-                else if (requiredVersion.Major < actualVersion.Major && !Config.ShouldLoadOutdatedPlugins)
+                else if (requiredVersion < actualVersion && !Config.ShouldLoadOutdatedPlugins)
                 {
-                    Log.Error($"You're running an older version of {plugin.Name} ({plugin.Version.ToString(3)})! " +
-                              $"Its Required Major version is {requiredVersion.Major}, but excepted {actualVersion.Major}. ");
+                    Log.Error($"You're running an older version of {plugin.Name} ({plugin.Version})! " +
```

Comment:
This error is painfully in need of some grammar checking, it should be:

"{plugin.Name} requires EXILED version {requiredVersion}, however your current version ({actualVersion}) is significantly newer. This may cause issues with the plugin, so it will not be loaded. To change this, set should_load_outdated_plugins to true."